includes/api/ApiViewDumps.php

```diff
@@ -54,21 +54,21 @@ public function execute() {
 
 				$buildResults[] = [
 					'filename' => $dump->dumps_filename,
-					'link' => $this->getDownloadUrl( $config, $dump ),
+					'link' => $this->getUrl( $config, $dump ),
```

Comment:
Use `getDownloadUrl()` again, since only one function now.

```suggestion
					'link' => $this->getDownloadUrl( $config, $dump ),
```